improve readme, improve .gitattributes

nathanfranke opened the pull request:
Fixes https://github.com/Poing-Studios/godot-admob-editor/issues/33

How it looks:
![image](https://user-images.githubusercontent.com/14253836/177065844-07825020-d71e-43a8-b71b-2c1a45aadf5b.png)

gumaciel reviewed (commented):
Please change the location of this image `addons/admob/assets/asset-library.png`

Put out of the "assets" folder for the user not to download this image when using the plugin.

Suggestions:

1. Create an "images" folder and add this image
2. Consider adding on https://imgur.com/ and put the link

nathanfranke commented on README.md:
```diff
@@ -1,67 +1,98 @@
-2. Enable the Plugin: https://docs.godotengine.org/pt_BR/stable/tutorials/plugins/editor/installing_plugins.html#enabling-a-plugin
+Godot Asset Library (recommended):
+1. Find the AdMob plugin by `poing.studios` \
+   <img height=100 src="https://raw.githubusercontent.com/Poing-Studios/godot-admob-editor/main/addons/admob/assets/asset-library.png">
```
It wouldn't work because that would point to `https://github.com/Poing-Studios/godot-admob-editor/addons/admob/assets/asset-library.png` which isn't a valid image.

nathanfranke commented:
All good suggestions, thanks! As for the image, I was going to put it in a separate folder but then I remembered that Godot would try to include that with the installation. See here:
![image](https://user-images.githubusercontent.com/14253836/177073108-7b4f321f-7d2a-4d4b-bc8b-977734b8e2cf.png)

Instead I uploaded the video directly using the README editor. Just in case that stops working, I attached another copy here. I didn't want to rely on a separate service to host the image, although it probably doesn't matter. :slightly_smiling_face: 

![asset_library](https://user-images.githubusercontent.com/14253836/177073028-b481ce6c-a77f-4573-80cd-77fb89f9dc4c.png)

gumaciel commented:
> All good suggestions, thanks! As for the image, I was going to put it in a separate folder but then I remembered that Godot would try to include that with the installation. See here: ![image](https://user-images.githubusercontent.com/14253836/177073108-7b4f321f-7d2a-4d4b-bc8b-977734b8e2cf.png)

Yes, but I just made this change: https://github.com/Poing-Studios/godot-admob-editor/issues/36

That way godot will only try to download what is actually inside the `assets/` folder, which is the content of our plugin

Read more here: https://github.com/godotengine/godot-proposals/issues/3648

I didn't want to add the image inside the `assets/` folder because the user will download an image that won't be useful for his project

nathanfranke commented:
Great! I think I will try that on my own projects too!

nathanfranke commented:
- Added the image to static, tested and the zip only contains addons :tada: 
- Also good news! Looks like relative paths actually do work in README! I updated the PR to utilize that.

gumaciel commented:
Thank you so much! Merged

gumaciel reviewed (commented):
Thank you!
